client/src/SubmitBidForm.js

```diff
@@ -4,7 +4,7 @@ import { get, post } from "./TenderClient";
 import "./SubmitBidForm.css";
 import Logo from "./assets/images/CTY-logo-rectangle.png";
 
-const SubmitBidForm = () => {
+const SubmitBidForm = ({ onBidSubmitted }) => {
```

Comment:
What are you going to use `onBidSubmitted` for?

Reply:
sorry was suppose to remove that. I will get rid of it